cli/cmd/resource_run_cmd.go

```diff
@@ -63,7 +63,7 @@ func init() {
 
 	runCmd.Flags().StringVarP(&runParams.DefinitionFile, "file", "f", "", "path to the definition file")
 	runCmd.Flags().StringVar(&runParams.ID, "id", "", "id of the resource to run")
-	runCmd.Flags().StringVarP(&runParams.EnvID, "environment", "e", "", "environment file or ID to be used")
+	runCmd.Flags().StringVarP(&runParams.VarsID, "vars", "", "", "variable set file or ID to be used")
```

Comment:
Does it make sense to be `variableset` here?
```suggestion
	runCmd.Flags().StringVarP(&runParams.VarsID, "variableset", "", "", "variable set file or ID to be used")
```

Reply:
As we were using a shorthand for environments, I thought we could continue using the same approach